src/engraving/rendering/dev/measurelayout.cpp

```diff
@@ -1233,16 +1233,15 @@ void MeasureLayout::layoutMeasureElements(Measure* m, LayoutContext& ctx)
                     MMRest* mmrest = toMMRest(e);
                     // center multimeasure rest
                     double d = ctx.conf().styleMM(Sid::multiMeasureRestMargin);
-                    double w = x2 - x1 - 2 * d;
                     bool headerException = m->header() && s.prev() && !s.prev()->isStartRepeatBarLineType();
                     if (headerException) { //needs this exception on header bar
-                        x1 = s1 ? s1->x() + s1->width() : 0;
-                        w = x2 - x1 - d;
+                        x1 = s.x() - ctx.conf().styleMM(Sid::noteBarDistance);
```

Comment:
I'd probably add a small comment to explain why we're using noteBarDistance here as it may look strange for someone who doesn't know. You can also do that in a next PR :)